cartography/cli.py

```diff
@@ -112,6 +112,23 @@ def _build_parser(self):
                 'respects the AWS CLI/SDK environment variables and does not override them.'
             ),
         )
+        parser.add_argument(
+            '--crxcavator-api-base-uri',
+            type=str,
+            default=None,
+            help=(
+                'Base URI for the CRXcavator API. Required if you are using the CRXcavator intel module. Ignored '
+                'otherwise.'
+            ),
+        )
+        parser.add_argument(
+            '--crxcavator-api-key-env-var',
+            type=str,
+            default=None,
+            help=(
+                'Key for the CRXcavator API. Required if you are using the CRXcavator intel module. Ignored otherwise.'
```

Comment:
How about: "Name of environment variable that contains your CRXcavator API key. Required if you are using the CRXcavator intel module. Ignored otherwise."?